acceptance-tests/bbl/upgrade_test.go

```diff
@@ -94,7 +93,7 @@ var _ = Describe("Upgrade", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	It("is able to upgrade from an environment bbl'd up with an older version of bbl", func() {
+	FIt("is able to upgrade from an environment bbl'd up with an older version of bbl", func() {
```

Comment:
Looks like there's still a test in focus here

Reply:
the Fit is a mistake and will solve it.
the other changes are due to the fact that #549 is not merged yet